poi/poi_types.xml

```diff
@@ -819,7 +819,10 @@
 		</poi_category>
 		<poi_category name="finance" default_tag="finance">
 			<poi_type name="bank" tag="amenity" value="bank"/>
-			<poi_type name="atm" tag="amenity" value="atm"/>
+			<poi_filter name="atm">
+				<poi_type name="atm_standalone" tag="amenity" value="atm"/>
+				<poi_type name="atm_at_building" tag="atm" value="yes"/>
```

Comment:
atm=yes used not only with buildings so I disagree with name http://overpass-turbo.eu/s/a5K